spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/util/Logging.scala

```diff
@@ -0,0 +1,80 @@
+package com.datastax.spark.connector.util
+
+import org.slf4j.{Logger, LoggerFactory}
+
+/**
+  * Utility trait for classes that want to log data. Creates a SLF4J logger for the class and allows
+  * logging messages at different levels using methods that only evaluate parameters lazily if the
+  * log level is enabled.
+  *
+  *
+  * This is a copy of what Spark Previously held in org.apache.spark.Logging. That class is
+  * now private so we will expose similar functionality here.
+  */
+trait Logging {
+  // Make the log field transient so that objects with Logging can
+  // be serialized and used on another machine
+  @transient private var log_ : Logger = null
```

Comment:
I know Spark did it this way, but I'm strongly against underscores following the argument - can you please put the underscore before the name?